src/redux/actions.test.js

```diff
@@ -8,6 +8,6 @@ describe("actions", () => {
     expect(actualAction.type).toBe(types.LOG_CALL);
     expect(actualAction.payload).toBeDefined();
     expect(actualAction.payload.districtId).toBe(districtId);
-    expect(actualAction.payload.timestamp).toBeCloseTo(Date.now(), -2);
+    expect(actualAction.payload.timestamp).toBeCloseTo(Date.now(), -1);
```

Comment:
If you are concerned that a factor of 10 reduction in precision is too much an alternative would be to subtract time from Date.now() so that they are closer.